academy/4-ibc/hermes-relayer.md

```diff
@@ -216,165 +209,219 @@ You can check the CLI commands with `hermes -h`. The Hermes CLI offers help for
+First, query packet commitments on **checkersa**:
 
 ```sh
-$ hermes -c simple_config.toml query packet commitments ibc-0 transfer channel-2
+$ hermes query packet commitments --chain checkersa --port transfer --channel channel-1
 ```
 
-You can see that there is one packet.
+You can see that there is one packet:
+
+```
+SUCCESS PacketSeqs {
+    height: Height {
+        revision: 0,
+        height: 2382,
+    },
+    seqs: [
+        Sequence(
+            1,
+        ),
+    ],
+}
+```
 
 You can also query for unreceived packets:
 
 ```sh
-$ hermes -c simple_config.toml query packet unreceived-packets ibc-1 transfer channel-5
+$ hermes query packet pending --chain checkersb --port transfer --channel channel-1
+```
+
+The output should be similar to:
+
 ```
+UCCESS Summary {
+    src: PendingPackets {
+        unreceived_packets: [],
+        unreceived_acks: [],
+    },
+    dst: PendingPackets {
+        unreceived_packets: [
+            Sequence(
+                1,
+            ),
+        ],
+        unreceived_acks: [],
+    },
+}
+```
+
+There you can observe an unreceived packet.
 
 <HighlightBox type="note">
 
-You can get the `connection_id` and `channel_id` for **ibc-1** in the output of the `hermes create connection` and `hermes create channel` commands.
+You can get the `connection_id` and `channel_id` for **checkersb** in the output of the `hermes create connection` and `hermes create channel` commands.
 
 </HighlightBox>
 
-If you check the balances again, you should only see a change for **ibc-0**. You should see no change in the balance of `testkey` on **ibc-1** because the transfer is initiated but it is not relayed yet.
+If you check the balances again, you should only see a change for **checkersa**. You should see no change in the balance of `bob` on **checkersb** because the transfer is initiated but it is not relayed yet.
 
-Now submit the `RecvPacket` message to **ibc-1**:
+Now submit the `RecvPacket` message to **checkersb**:
 
 ```sh
-$ hermes -c simple_config.toml tx raw packet-recv ibc-1 ibc-0 transfer channel-2
+$ hermes tx packet-recv --dst-chain checkersb --src-chain checkersa --src-port transfer --src-channel channel-1
 ```
 
-Send an acknowledgement to **ibc-0**:
+In case of success, you will see an output like:
+
+```
+SUCCESS [
+    SendPacket(
+        SendPacket - seq:1, path:channel-1/transfer->channel-1/transfer, toh:0-3368, tos:Timestamp(NoTimestamp)),
+    ),
+]
+```
+
+Send an acknowledgment to **checkersa**:
 
 ```sh
-$ hermes -c simple_config.toml tx raw packet-ack ibc-0 ibc-1 transfer channel-5
+$ hermes tx packet-ack --dst-chain checkersa --src-chain checkersb --src-port transfer --src-channel channel-1
 ```
 
-Check the balances again. A new denom should appear because of our recent channel. As an exercise, transfer the tokens back to ibc-0.
+Check the balances again. A new denom should appear because of our recent transfer. As an exercise, transfer the tokens back to **checkersa**.
```

Comment:
```suggestion
Check the balances again. A new denom should appear because of the recent transfer. As an exercise, transfer the tokens back to **checkersa**.
```